core/src/main/java/org/apache/iceberg/catalog/BaseSessionCatalog.java

```diff
@@ -62,6 +62,11 @@ public <T> T withContext(SessionContext context, Function<Catalog, T> task) {
     return task.apply(asCatalog(context));
   }
 
+  public void multiTableCommit(SessionContext context, List<TableCommit> commits) {
```

Comment:
This doesn't really fit with the phrasing of the other methods in this class, like `loadTable`. What about something like `commitTables`? That isn't great, but `multiTableCommit` uses "commit" as a noun rather than a verb.

Reply:
`commitTables` sounds better, let's go with that for now